docs/sources/tempo/troubleshooting/send-traces/max-trace-limit-reached.md

```diff
@@ -53,13 +53,7 @@ msg="pusher failed to consume trace data" err="rpc error: code = FailedPrecondit
 msg="pusher failed to consume trace data" err="rpc error: code = ResourceExhausted desc = RATE_LIMITED: ingestion rate limit (15000000 bytes) exceeded while adding 10 bytes"
 ```
 
-You'll also see the following metric incremented. The `reason` label on this metric will contain information about the refused reason.
```

Comment:
Should we replace this with anything?

Reply:
tempo_discarded_spans_total still exists. we just remove the "InternalError" reason

Reply:
Oh! My bad for misunderstanding. I'll update this doc then. This was the only place in the Tempo docs that we mentioned tempo_discarded_spans_total.

Reply:
In that case, this information is probably still okay and I can close this PR.